content/_help/verify-your-identity/phone-number-and-phone-plan-in-your-name._en.md

```diff
@@ -1,22 +1,16 @@
 ---
 layout: help
-title: "Phone number and phone plan in your name "
+title: Verify your phone number
 category: verify-your-identity
-permalink: /help/verify-your-identity/phone-number-and-phone-plan-in-your-name/
+permalink: /help/verify-your-identity/phone-number/
 order: 5
 redirect_from:
   - /en/help/verify-your-identity/phone-number-and-phone-plan-in-your-name/
```

Comment:
Per the failing build, if we're updating the URL, we should add a redirect so that the old URL continues to work for users who had previously bookmarked the page or partners who have referenced it in their own documentation.

```suggestion
permalink: /help/verify-your-identity/phone-number/
order: 5
redirect_from:
  - /help/verify-your-identity/phone-number-and-phone-plan-in-your-name/
  - /en/help/verify-your-identity/phone-number/
  - /en/help/verify-your-identity/phone-number-and-phone-plan-in-your-name/
```

Reply:
Let's add this to the handbook

Reply:
> Per the failing build, if we're updating the URL, we should add a redirect so that the old URL continues to work for users who had previously bookmarked the page or partners who have referenced it in their own documentation.

Let's add this to the handbook for future editing.